code/modules/mob/living/carbon/human/life.dm

```diff
@@ -75,6 +75,10 @@
 
 		handle_medical_side_effects()
 
+		if(life_tick % 3 == 1)	//Upadated every 3 life ticks, lots of for loops in this
```

Comment:
You can get rid the `== 1` the modulo always return 0 or 1 for this case

Reply:
Fixed.